my-crccheck.yml

```diff
@@ -79,17 +78,28 @@ spec:
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-  name: web-crccheck-ingress
+  name: ingress-service-crccheck
   namespace: mike-space
+  labels:
+    typeserv : mytype 
 spec:
   rules:
-    - host: hello-world.info
+    - host: www.myingress.com
```

Comment:
Have you changed /etc/hosts?

Reply:
Yep!!! $ sudo nano /etc/hosts 

127.0.0.1	                localhost
127.0.1.1	                CMDB-80317
192.168.99.128	www.myingress.com
192.168.99.124       hello-world.info